R/profile_wrapper.R

```diff
@@ -1,27 +1,42 @@
+#' Run [r4ss::profile] based on `model_settings`
+#'
 #' Generate likelihood profiles
 #' To be called from the run_diagnostics function after creating
 #' the model settings using the get_settings function.
 #' 
 #'
+#' @seealso The following functions interact with `profile_wrapper`:
+#' * [run_diagnostics]: calls `profile_wrapper`
+#' * [r4ss::profile]: the workhorse of `profile_wrapper` that does the parameter profiles
+#'
+#'
 #' @template mydir
 #' @template model_settings
 #' 
 #' @author Chantel Wetzel.
+#' @return
+#' Nothing is explicitly returned from `profile_wrapper`.
+#' The following objects are saved to the disk.
 #' @export
 
 profile_wrapper <- function(mydir, model_settings){
 
+  # Add the round_any function from the plyr package to avoid conflicts between
+  # plyr and dplyr.
+  round_any <-  function(x, accuracy, f = round){
+    f(x / accuracy) * accuracy}
+
 	OS <- "Mac" # don't know the version$os info for Mac
```

Comment:
you should be able to delete the determination of OS and checking the exe in lines 29 to 48 because these things should now all be taken care of by `r4ss::profile()` which calls `run()` which calls `check_exe()`.